.github/workflows/changelog.yml

```
@@ -0,0 +1,33 @@
+    branches: [ main ]
+  workflow_dispatch:
+
+jobs:
+  generate-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)